### ISSUE TITLE
Create Interface for Non-Availability / Judge Algorithm

### ISSUE THREAD

**Issue by lomaxap** [department-of-veterans-affairs/caseflow#12005]

**Comment by jimruggiero** [comment]
@lomaxap Is this issue still valid and/or needed? If yes, can you please add a user story and some context? If this is issue is no longer required can you please close it? Thanks.

For now, moving this to the Backlog. (cc'ing @pshahVA and @nicholer for awareness)

**Comment by pshahVA** [comment]
Yes, this is something that we've identified as a need to clean up the scheduling altorithm, instead of uploading the spreadsheet.  This work has not been prioritized or even discussed with out stakeholders.
